transforms/README.md

```diff
@@ -0,0 +1,26 @@
+# @fluentui-react-native/transforms
+
+This package supplies transforms that help with refactoring FURN code. The transforms are passed into [jscodeshift](https://github.com/facebook/jscodeshift) to transform code.
+
+## Usage
+
+1. Install jscodeshift using the instructions [here](https://github.com/facebook/jscodeshift#install)
+2. Run the transforms using the following command:
+
+```cli
+jscodeshift -t <path to transform file> --parser=tsx --extensions=tsx <path to file to be transformed>
+```
+
+For example
+
+```cli
+jscodeshift -t transforms/src/button-v0-to-v1.ts --parser=tsx --extensions=tsx apps/fluent-tester/src/TestComponents/Button/deprecated
+```
```

Comment:
How would a downstream client use this? Would they clone our repo temporarily to run it?

Reply:
They could get the code by either copying it or downloading the package. I guess in that case it probably doesn't make sense to run TSC on the transform files.

Reply:
Hm I guess I can make a bin that will execute the script...

Reply:
I think I'd rather figure that out separately. Right now the code can be executed in the repo or by copying the transform file out into its own folder, so you don't really need to clone the repo in order to run the transform, and I've mostly been applying it locally so far.